servicetalk-examples/grpc/protoc-options/README.adoc

```diff
@@ -0,0 +1,14 @@
+== ServiceTalk gRPC protoc options
+
+Using the same interface as the "Hello World" examples, demonstrate servicetalk-grpc-protoc plugin options.
+
+For example here is one of the options demonstrated:
+
+[source,gradle]
+----
+task.plugins {
+  servicetalk_grpc {
+    option 'typeNameSuffix=St'
```

Comment:
Consider adding a comment here, describing what this option does and referencing the servicetalk-grpc-protoc/README.adoc file for more information.

Also, we need to update servicetalk-examples/docs/modules/ROOT/pages/_partials/nav-versioned.adoc and servicetalk-examples/docs/modules/ROOT/pages/grpc/index.adoc file to include this example.

Reply:
done